### ISSUE TITLE
grpc_php_plugin cannot be built as in quick start/tutorial for PHP

### ISSUE THREAD

**Issue by justin-wilxite** [grpc/grpc#25350]
Tried following the tutorial for PHP: https://grpc.io/docs/languages/php/basics/

`$ cd grpc && git submodule update --init`
`$ make grpc_php_plugin`
`make: *** No rule to make target 'grpc_php_plugin'.  Stop.`

I am ultimately trying to compile my own prototype file. I tried this:
`protoc --php_out=. --grpc_out=. test.proto`

But this does not generate the client stub classes that I need. So I clearly need the grpc php plugin but I cannot compile it.

If I try to use the included route_guide_proto_gen.sh:
```
./../../../bins/opt/grpc_php_plugin: program not found or is not executable
Please specify a program using absolute path or make sure the program is available in your PATH system variable
--grpc_out: protoc-gen-grpc: Plugin failed with status code 1.
```

I tried using the master branch as suggested in a previous issue about this exact problem, but for me it did not work. #22108

### What version of gRPC and what language are you using?
grpc 1.7.0
protobuf 3.12.4

### What operating system (Linux, Windows,...) and version?
Linux

**Comment by benammann** [comment]
Just had the same issue. Using an older version works for me:

```
RUN cd /tmp && git clone -b v1.34.1 --depth 1 https://github.com/grpc/grpc && cd grpc && git submodule update --init && make grpc_php_plugin
```

**Comment by justin-wilxite** [comment]
> Just had the same issue. Using an older version works for me:
> 
> ```
> RUN cd /tmp && git clone -b v1.34.1 --depth 1 https://github.com/grpc/grpc && cd grpc && git submodule update --init && make grpc_php_plugin
> ```

Thank you! That version worked for me too

**Comment by stanley-cheung** [comment]
We probably should update the docs. You can build it with bazel now:

```
./tools/bazel build @com_google_protobuf//:protoc
./tools/bazel build src/compiler:grpc_php_plugin
```

**Comment by 4br3mm0rd** [comment]
It could be nice if it was added to the compiled releases of protoc as well :)

**Comment by jtattermusch** [comment]
Note that https://github.com/grpc/grpc/tree/master/src/php#grpc_php_plugin-protoc-plugin also need updating.

**Comment by chalin** [comment]
This is reporting a problem with grpc.io docs, for which there is the following issue: https://github.com/grpc/grpc.io/issues/664.

@jtattermusch @stanley-cheung - can you close this issue? We'll track it's resolution at https://github.com/grpc/grpc.io/issues/664.

**Comment by jtattermusch** [comment]
Fixed by #25544 and https://github.com/grpc/grpc.io/pull/684.
